php/Dockerfile

```diff
@@ -46,13 +46,26 @@ RUN \
         --no-suggest --optimize-autoloader --classmap-authoritative
 ENV PATH "${PATH}:/root/.composer/vendor/bin"
 
+# Create directories where Magento will write
```

Comment:
Is it really useful to pre-create these directories or is it just to change the owner ?